src/base/static/components/app.tsx

```diff
@@ -169,18 +168,44 @@ declare const Mapseed: any;
 // 'process' global is injected by Webpack:
 declare const process: any;
 
-interface State {
+export interface IMapViewport {
```

Comment:
Since we're exporting this interface, I thought it would be useful to start adopting the `I` syntax for interfaces to make it clearer when an interface is being imported into another module.

Reply:
I considered using the `I` prefix, and decided not to do it based on some guidelines I found here: https://angular.io/guide/styleguide#interfaces and here: https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript/41967120#41967120

But I don't have a strong opinion on it. So feel free to stick with what you think is more comfortable. I found a full flame war on the topic here: https://github.com/Microsoft/TypeScript-Handbook/issues/121

Reply:
Hmm, well I guess I'm not surprised it's controversial. Within a module it seems like overkill, but when exporting interfaces I thought it would make things clearer. But that's based on my one-day experience with TS.

Maybe I'll just leave it for now, and if it turns out to be silly we can revert the `I` syntax.